app/controllers/application.js

```diff
@@ -2,9 +2,11 @@ import Controller from '@ember/controller';
 import { inject as service } from '@ember/service';
 import { LOGOUT_URL, MAIN_SITE_URL } from '../constants/navbar';
 import { action } from '@ember/object';
+import { GITHUB_URL } from '../constants/url';
 
 export default class ApplicationController extends Controller {
   @service router;
+  url = GITHUB_URL;
```

Comment:
Can we name it better to quickly understand what this URL is? Naming it same as the value can also work i.e. `GITHUB_URL`

Reply:
Corrected it.